.editorconfig

```diff
@@ -8,7 +8,7 @@ insert_final_newline = true
 charset = utf-8
 trim_trailing_whitespace = true
 
-[*.{c,h}]
+[*.{c,h,eval,rules}]
```

Comment:
This had me confused. What is *.eval? Maybe list the full file names here. And make it a separate stanza from the C-code related things.